packages/perseus/src/util/parse-perseus-json/perseus-parsers/measurer-widget.ts

```diff
@@ -17,7 +17,14 @@ import type {Parser} from "../parser-types";
 export const parseMeasurerWidget: Parser<MeasurerWidget> = parseWidget(
     constant("measurer"),
     object({
-        image: parsePerseusImageBackground,
+        // The default value for image comes from measurer.tsx.
```

Comment:
Thanks for adding these comments in!  I think they'll be really helpful going forward.